packages/api/cms-api/src/sentry/sentry.module.ts

```diff
@@ -0,0 +1,41 @@
+import { DynamicModule, Module } from "@nestjs/common";
+import { APP_INTERCEPTOR } from "@nestjs/core";
+
+import { SentryInterceptor, SentryInterceptorOptions } from "./sentry.intercepor";
+
+type SentryNodeOptions = Omit<import("@sentry/node").NodeOptions, "dsn" | "environment"> & {
+    dsn: string;
+    environment: string;
+};
+
+type Options = SentryNodeOptions & SentryInterceptorOptions;
+
+@Module({})
+export class SentryModule {
+    static async forRootAsync(options: Options): Promise<DynamicModule> {
+        const Sentry = await import("@sentry/node");
+
+        if (!Sentry) {
+            return {
+                module: SentryModule,
+            };
+        }
```

Comment:
Is checking for undefined necessary? If yes, we should throw an error instead

Reply:
Removed